packages/lib-classifier/src/components/Classifier/components/SubjectViewer/helpers/isDataSeriesHighlighted/isDataSeriesHighlighted.js

```diff
@@ -1,4 +1,4 @@
-export default function isDataSeriesHighlighted (highlightedSeries = [], seriesIndex = 0) {
-  const [highlighted] = (highlightedSeries?.length > 0) ? Object.values(highlightedSeries[seriesIndex]) : [true]
-  return highlighted
+export default function isDataSeriesHighlighted ({ highlightedSeries: highlightedSeries = [], seriesOptions: seriesOptions = {} } = {}) {
+  if (highlightedSeries.length > 0 && seriesOptions.label) return highlightedSeries.includes(seriesOptions.label)
+  return true
```

Comment:
I think `return true` is causing issues when attempting to uncheck both series, should be `return false`? Though I'm not sure in practice when one would ever want to uncheck both series.

Reply:
This is now fixed. I do want this to default to true, but the problem was having the default parameter for `highlightedSeries`. The reason is that this helper is used both by the `VariableStarViewer` and underlying `ScatterPlotViewer` on its own, so if you have multiple data series in the base scatter plot, we want them to all have fill colors since the highlight state is not toggleable there.